src/browser/rollbar.js

```diff
@@ -87,6 +91,9 @@ Rollbar.log = function() {
 Rollbar.prototype.debug = function() {
   var item = this._createItem(arguments);
   var uuid = item.uuid;
+  if (this._sameAsLastError(item)) {
```

Comment:
Even if we did, the client would then have to track the lastError, but then lastError couldn't be a property on the top level instance, unless we returned lastError from `client.log` or converted lastError to a function